src/components/ButtonComponents/SpecialButtons/SpecialButton.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 
-const SpecialButton = () => {
+const SpecialButton = props => {
```

Comment:
This is a valid way of passing in props, though you could enhance the readability and DRYness of your code in the future by destructuring your props (e.g. props could be {test3}).